ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/tasks/pre_workload.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- name: virtualenv path to use for pip
+  set_fact:
+    pip_path: "{{ ansible_python_interpreter | dirname }}"
+
+
+- name: Install lxml into virtualenv
+  command:  "{{ pip_path }}/pip install lxml"
```

Comment:
Use `pip` module. And make sure to pin the version of `lxml`.

Reply:
Resolved